vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/GridViewIT.java

```diff
@@ -693,6 +693,23 @@ public void gridWithFiltering() {
         waitUntil(driver -> grid.getCell(0, 0).getText().contains("Person 6"));
     }
 
+    @Test
+    public void gridWithFiltering_scrolledToEnd() {
```

Comment:
The test name could use an expectation, and filtering and scrolling run in different order. Maybe `scrollToEnd_filter_rowsUpdated`?

Reply:
Done